src/Nethermind/Nethermind.Taiko/TaikoAnchorTxReleaseSpec.cs

```diff
@@ -0,0 +1,152 @@
+using Nethermind.Core;
+using Nethermind.Core.Specs;
+using Nethermind.Int256;
+
+namespace Nethermind.Taiko;
+
+public class TaikoAnchorTxReleaseSpec(IReleaseSpec parent, Address? eip1559FeeCollector) : IReleaseSpec
+{
+    private readonly IReleaseSpec _parent = parent;
+    private readonly Address? eip1559FeeCollector = eip1559FeeCollector;
+
+    public string Name => _parent.Name;
+
+    public long MaximumExtraDataSize => _parent.MaximumExtraDataSize;
+
+    public long MaxCodeSize => _parent.MaxCodeSize;
+
+    public long MinGasLimit => _parent.MinGasLimit;
+
+    public long GasLimitBoundDivisor => _parent.GasLimitBoundDivisor;
+
+    public UInt256 BlockReward => _parent.BlockReward;
+
+    public long DifficultyBombDelay => _parent.DifficultyBombDelay;
+
+    public long DifficultyBoundDivisor => _parent.DifficultyBoundDivisor;
+
+    public long? FixedDifficulty => _parent.FixedDifficulty;
+
+    public int MaximumUncleCount => _parent.MaximumUncleCount;
+
+    public bool IsTimeAdjustmentPostOlympic => _parent.IsTimeAdjustmentPostOlympic;
+
+    public bool IsEip2Enabled => _parent.IsEip2Enabled;
```

Comment:
We should make a note to refactor these `IsEipXXXEnabled` flags to something more reasonable.